apps/web/src/components/generic/SearchBar.tsx

```diff
@@ -1,8 +1,11 @@
+import { useOccupationSearchContext } from '../planning/Occupation';
+
 interface SearchBarProps {
   placeholderText: string;
 }
 
 export const SearchBar = ({ placeholderText = 'Search' }: SearchBarProps) => {
+  const searchValue = useOccupationSearchContext();
```

Comment:
Nope, cannot use a non-generic thing in a generic component. Kindly refer Peter's PR